debian.Dockerfile

```diff
@@ -0,0 +1,93 @@
+    tar xvf gnutls-3.6.9.tar.xz && \
+    cd gnutls-3.6.9 && \
+    ./configure --prefix=/usr/local/gnutls/3_6_9 && \
+    make && make install && \
+    ln -s /usr/local/gnutls/3_6_9/bin/* /usr/local/bin/ && \
+    ln -s /usr/local/gnutls/3_6_9/include/gnutls /usr/local/include/ && \
+    ln -s /usr/local/gnutls/3_6_9/lib/* /usr/local/lib/
+WORKDIR /tmp/build/gvm-libs
+RUN set -x && \
+  git clone https://github.com/root-secure/gvm-libs.git && \
+  cd gvm-libs && \
+  git fetch && git checkout awn-gvm-libs-21.4.4 && \
+  mkdir build && cd build && \
+  cmake -DCMAKE_INSTALL_PREFIX=${LIB_INSTALL_PREFIX} .. && make && make install
+WORKDIR /tmp/build/openvas
+COPY . .
+RUN set -x && \
+  mkdir build && cd build && \
+  cmake -DCMAKE_INSTALL_PREFIX=${LIB_INSTALL_PREFIX} .. && make && make install
+RUN set -x && \
+  # create directories
+  mkdir -p ${OPENVAS_DEB_BUILD_DIR}/openvas-21.4.*/usr \
+  ${OPENVAS_DEB_BUILD_DIR}/openvas-21.4.*/usr/bin \
+  ${OPENVAS_DEB_BUILD_DIR}/openvas-21.4.*/usr/etc/openvas \
+  ${OPENVAS_DEB_BUILD_DIR}/openvas-21.4.*/usr/lib \
+  ${OPENVAS_DEB_BUILD_DIR}/openvas-21.4.*/usr/sbin \
+  ${OPENVAS_DEB_BUILD_DIR}/openvas-21.4.*/usr/share/doc/openvas-scanner/redis_config_examples \
+  ${OPENVAS_DEB_BUILD_DIR}/openvas-21.4.*/usr/share/man \
+  ${OPENVAS_DEB_BUILD_DIR}/openvas-21.4.*/usr/share/man/man1 \
+  ${OPENVAS_DEB_BUILD_DIR}/openvas-21.4.*/usr/share/man/man8 \
+  ${OPENVAS_DEB_BUILD_DIR}/openvas-21.4.*/usr/share/openvas \
+  ${OPENVAS_DEB_BUILD_DIR}/openvas-21.4.*/usr/var/lib/openvas/gnupg \
+  ${OPENVAS_DEB_BUILD_DIR}/openvas-21.4.*/usr/log/gvm && \
+  # Create symlinks
+  ln -s /etc /usr/etc && \
+  ln -s /sbin /usr/sbin && \
+  # copy files
+  cp /usr/bin/openvas-* ${OPENVAS_DEB_BUILD_DIR}/openvas-21.4.*/usr/bin/ && \
+  cp -r /usr/etc/openvas/* ${OPENVAS_DEB_BUILD_DIR}/openvas-21.4.*/usr/etc/openvas/ && \
+  cp /usr/lib/libopenvas*.so.21.4.* ${OPENVAS_DEB_BUILD_DIR}/openvas-21.4.*/usr/lib/ && \
+#  cp /usr/sbin/greenbone-* ${OPENVAS_DEB_BUILD_DIR}/openvas-21.4.*/usr/sbin/ && \
+  cp /usr/sbin/openvas ${OPENVAS_DEB_BUILD_DIR}/openvas-21.4.*/usr/sbin/ && \
+#  cp /usr/share/doc/openvas-scanner/redis_config_examples/redis*.conf ${OPENVAS_DEB_BUILD_DIR}/openvas-21.4.*/usr/share/doc/openvas-scanner/redis_config_examples/ && \
+  cp /usr/share/man/man1/openvas-*.1 ${OPENVAS_DEB_BUILD_DIR}/openvas-21.4.*/usr/share/man/man1/ && \
+  cp /usr/share/man/man8/greenbone-nvt-sync.8 ${OPENVAS_DEB_BUILD_DIR}/openvas-21.4.*/usr/share/man/man8/ && \
+  cp /usr/share/man/man8/openvas.8 ${OPENVAS_DEB_BUILD_DIR}/openvas-21.4.*/usr/share/man/man8/ && \
+  # create original archive
+  cd ${OPENVAS_DEB_BUILD_DIR} && \
+  tar -czvf openvas_21.4.4.orig.tar.gz openvas-21.4.*
+COPY Debian/openvas ${OPENVAS_DEB_BUILD_DIR}/openvas-21.4.*
+RUN set -x && \
+  cd ${OPENVAS_DEB_BUILD_DIR}/openvas-21.4.* && \
+  debuild -us -uc
+COPY Debian/openvas-scanner ${OPENVAS_SCANNER_DEB_BUILD_DIR}/openvas-scanner
```

Comment:
I don't find a good reason to have two separate debian packages.
I am thinking of moving `Debian/openvas-scanner` into `Debian/openvas` to combine them together.
The `Debian/openvas-scanner` is light weight package and carries few config files, that we can combine the with `Debian/openvas`.
Here is the packages size:
`➜  openvas git:(AWN-84747) ✗ ll
...
-rw-r--r--   1 shanmukh.basavarajappa  staff    20K 13 Apr 17:31 openvas-scanner_21.4.4_amd64.deb
-rw-r--r--   1 shanmukh.basavarajappa  staff   207K 13 Apr 17:30 openvas_21.4.4_amd64.deb
...`